### PR TITLE
add description fields, upload button, submit button, and new page de…

### PR DESCRIPTION
**Pull request by katch00**
Fields and buttons for the uploads page. there's that weird error still but I get no other errors and the website runs :)

### CODE REVIEW

**Comment by indhu-gunda** [comment]
Can you add a screenshot of the UI changes? (just so its quicker for us to see)

**Comment by katch00** [comment]
capture
Hopefully this works

File: frontend/src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -174,8 +175,9 @@ export default function Dashboard() {
                   Upload Artwork
                 </Typography>
                 <Typography variant="body1" gutterBottom>
-                  Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
+                  Here you can upload artwork of your own! Please be sure to include you name and a very detailed description of your artwork. If you are unsure of what qualifies as a good description, we've provided some links below to some resources that can help guide you!
```
Comment (isabelsang) [inline frontend/src/components/dashboard/Dashboard.js:178]: nit: your name instead of you name

**Comment by indhu-gunda** [comment]
The form looks good! It might seem a little sparse though since all the fields are to the left. I think an option could be to make the fields extend horizontally (similar to your mock on the design doc) or put the first and last name in the same row and make the description field bigger. This can be done in another PR though.

**Pull request merged**